include/deal.II/base/mpi.h

```diff
@@ -1069,6 +1069,7 @@ namespace Utilities
                   const std::vector<T2> &recv_buffer);
     };
 
+
```

Comment:
Can you add another empty line as we typically use three between different classes?